src/integrationTest/java/uk/gov/hmcts/reform/feature/config/FlywayIntegrationConfig.java

```diff
@@ -1,12 +1,36 @@
 package uk.gov.hmcts.reform.feature.config;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.autoconfigure.AutoConfigureAfter;
 import org.springframework.boot.autoconfigure.flyway.FlywayAutoConfiguration;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.DependsOn;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.jdbc.datasource.DataSourceUtils;
+import org.springframework.jdbc.datasource.init.ScriptUtils;
+
+import javax.annotation.PostConstruct;
+import javax.sql.DataSource;
 
 @Configuration
 @AutoConfigureAfter(FlywayAutoConfiguration.class)
 @DependsOn({"flyway", "flywayInitializer"})
 public class FlywayIntegrationConfig {
+    @Autowired
+    private DataSource dataSource;
+
+    @PostConstruct
+    public void loadSqlData() {
+        //Clear data
+        executeSql("delete-exisiting-data.sql");
```

Comment:
aren't integration tests using testcontainers? ie, isn't the db always empty when tests start?

Reply:
I had a look at this one `@PostConstruct` are  invoked twice by Spring boot and hence `loadSqlData()` is executed twice due to which unique key constraints fails.I think it's fixed in Spring boot 2.0.2 RELEASE(Ensure test @PostConstructs are only called once) but FF4J doesn't support spring boot 2 atm.

Reply:
if that's a workaround then add a comment in code please